staging/src/k8s.io/api/admissionregistration/v1beta1/types.go

```diff
@@ -265,6 +265,11 @@ type WebhookClientConfig struct {
 	// the webhook's server certificate.
 	// Required.
 	CABundle []byte `json:"caBundle" protobuf:"bytes,2,opt,name=caBundle"`
+
+	// `timeout` is the number of seconds after which the webhook request times out.
```

Comment:
nit: the unit is specified by the user, not necessarily in seconds, see https://golang.org/pkg/time/#Duration.String. I would say "`timeout` is the duration after which the webhook request times out."

Reply:
sgtm